File: python/stack/array.stack.py
class Stack:
    def __init__(self):
        self.stack = []
        
    def push(self,data):
        self.stack.append(data)
        
    def empty(self):
        return len(self.stack) == 0
    
    def peek(self):
        return None if self.empty() else self.stack[len(self.stack) - 1]
    
    

File: js/stack/array.stack.js
class Stack{
    constructor(){
        this.stack = [];
    }


    push(data){
        this.stack.push(data)
    }

    empty(){
        return this.stack.length === 0
    }

    pop(){
        if(this.empty())
            return null;
        return  this.stack.pop()
    }

    peek(){
        if(this.empty())
            return null;
        return this.stack[this.stack.length - 1]
    }
}